AzureSDK.xcworkspace/contents.xcworkspacedata

```diff
@@ -49,9 +49,6 @@
    <Group
       location = "group:examples"
       name = "Examples">
-      <FileRef
-         location = "group:AzureSDKSmokeTest/AzureSDKSmokeTest.xcodeproj">
-      </FileRef>
```

Comment:
Why was this removed? It should appear under Examples in the AzureSDK workspace.

Reply:
I removed it from the AzureSDK workspace because it didn't appear to build in that workspace. You have to go into the AzureSDKSmokeTest directory, run `pod install` there to install the pods, and then open `AzureSDKSmokeTest.xcworkspace` and build it from there, because the pod search directory is relative to the workspace.

Reply:
We can revisit whether the smoke test belongs in the examples/ directory in a future PR. For now I am going to keep it there.